cmd/file_lint.go

```diff
@@ -57,16 +59,43 @@ func ParseSeverity(s string) Severity {
 	return SeverityWarn
 }
 
+func isOpenAPISpec(fileBytes []byte) bool {
+	var contents map[string]interface{}
+
+	// This marshalling is redundant with what happens
+	// in the linting command. There is likely an algorithm
+	// we could use to determine JSON vs YAML and pull out the
+	// openapi key without unmarshalling the entire file.
+	err := json.Unmarshal(fileBytes, &contents)
+	if err != nil {
+		err = yaml.Unmarshal(fileBytes, &contents)
+	}
```

Comment:
you can just use a YAML decoder, since JSON is also valid YAML.

Reply:
thanks, changed